src/transactions/feed/TransactionFeed.tsx

```diff
@@ -108,7 +109,11 @@ function TransactionFeed() {
   function renderItem({ item: tx }: { item: FeedTokenTransaction; index: number }) {
     switch (tx.__typename) {
       case 'TokenExchangeV2':
-        return <ExchangeFeedItem key={tx.transactionHash} exchange={tx} />
+        if (tx.type == 'SWAP_TRANSACTION') {
+          return <SwapFeedItem key={tx.transactionHash} exchange={tx} />
+        } else {
+          return <ExchangeFeedItem key={tx.transactionHash} exchange={tx} />
```

Comment:
i think it is worth asking Ellen whether we should keep both. from the way blockchain-api (should) work, the buy / sell CELO action should be now classified as a swap? i would advise to remove the existing ExchangeFeedItem but let's check with Ellen first

Reply:
I removed it after discussing it with @ellenwoconnor